docs/apps/qiskit.md

```diff
@@ -16,14 +16,15 @@ of circuits, pulses, and algorithms.
 
 Currently supported Qiskit versions:
 
-| Version | Module          | Puhti | Mahti | Notes           |
-| :------ | :-------------- | :---: | :---: | --------------- |
-| 0.43.2  | `qiskit/0.43.2` |   X   |   X   |                 |
-| 0.45.3  | `qiskit/0.45.3` |   X   |   X   |                 |
-| 1.0.2   | `qiskit/1.0.2`  |   X   |   X   | default version |
+| Version | Module          | Puhti | Mahti | LUMI | Notes           |
+| :------ | :-------------- | :---: | :---: | :---:| --------------- |
+| 0.43.2  | `qiskit/0.43.2` |   X   |   X   |      |                 |
+| 0.45.3  | `qiskit/0.45.3` |   X   |   X   |      |                 |
+| 1.0.2   | `qiskit/1.0.2`  |   X   |   X   |      | default version |
+| 1.1.1   | `qiskit/1.1.1`  |       |       |  X   | default version |
 
 Includes all the major Qiskit packages (Terra, Nature, Aer, etc.) and GPU support for the
-CUDA-accelerated simulation methods. The `qiskit/1.0.2` package includes the following qiskit plugins:
+CUDA-accelerated simulation methods. The `qiskit/1.0.2` and `qiskit/1.1.1` packages include the following qiskit plugins:
```

Comment:
Perhaps just say "GPU accelerated" and remove specific mention to cuda as we have LUMI now?